src/core_atmosphere/physics/mpas_atmphys_init.F

```diff
@@ -397,6 +410,55 @@ subroutine physics_init(dminfo,clock,configs,mesh,diag,tend,state,time_lev,diag_
 
  end subroutine physics_init
 
+!=================================================================================================================
+ subroutine init_physics_flags(state,f_qc,f_qr,f_qi,f_qs,f_qg,f_qoz,f_nc,f_ni,f_nifa,f_nwfa,f_nbca)
+!=================================================================================================================
+
+!input arguments:
+ type(mpas_pool_type),intent(in):: state
+
+!output arguments:
+ logical,intent(out):: f_qc,f_qr,f_qi,f_qs,f_qg,f_qoz
+ logical,intent(out):: f_nc,f_ni,f_nifa,f_nwfa,f_nbca
+
+!local pointers:
+ integer,pointer:: index_qc,index_qr,index_qi,index_qs,index_qg
+ integer,pointer:: index_ni
+
+!-----------------------------------------------------------------------------------------------------------------
+
+!initializes the logicals assigned to mixing ratios:
+ f_qc  = .false.
+ f_qr  = .false.
+ f_qi  = .false.
+ f_qs  = .false.
+ f_qg  = .false.
+ f_qoz = .false. !qoz is not defined in Registry.xml and f_qoz is initialized to false.
```

Comment:
Note: MPAS doesn't have ozone yet(?). 
Can it even be used for global modeling?